docs/packs/packs-format.md

```diff
@@ -66,6 +66,7 @@ The following fields are populated in the pack metadata:
 | `devEmail` | `List` | (Optional) List of emails to receive notification in case contributed pack files were modified. |
 | `certification` | `String` | (Optional) If the pack is certified the value of this fields should be `certified`. The allowed values are `certified` and `verified`. |
 | `itemPrefix` | `String` | (Optional) String to overwrite pack fields prefix. You can specify an alternative string instead of the default pack name enforced by the validation process. |
```

Comment:
minor update:
(Optional) String to overwrite the pack field prefix. You can specify an alternate string instead of the default pack name enforced by the validation process.

Reply:
```suggestion
| `itemPrefix` | `String` | (Optional) String to overwrite the pack field prefix. You can specify an alternate string instead of the default pack name enforced by the validation process. |
```